src/main/kotlin/no/digdir/informasjonsforvaltning/fdk_dataservice_harvester/controller/CatalogsController.kt

```diff
@@ -0,0 +1,31 @@
+package no.digdir.informasjonsforvaltning.fdk_dataservice_harvester.controller
+
+import no.digdir.informasjonsforvaltning.fdk_dataservice_harvester.generated.api.DcatApNoCatalogsApi
+import no.digdir.informasjonsforvaltning.fdk_dataservice_harvester.rdf.jenaTypeFromAcceptHeader
+import no.digdir.informasjonsforvaltning.fdk_dataservice_harvester.service.CatalogService
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Controller
+import javax.servlet.http.HttpServletRequest
+
+private val LOGGER = LoggerFactory.getLogger(CatalogsController::class.java)
+
+@Controller
```

Comment:
Hvorfor ikke RestController?

Reply:
'RestController' er en samling av 'Controller' og 'ResponseBody'. 'ResponseBody' sier at responsen skal serialiseres, til feks json, men alle responsene er allerede serialisert til rdf.

-> No need for 'ResponseBody' -> Bruker kun 'Controller'